scripts/lib/CIME/case/case_st_archive.py

```diff
@@ -413,7 +413,7 @@ def _archive_restarts_date_comp(case, casename, rundir, archive, archive_entry,
 #        logger.debug("suffix is {} ninst {}".format(suffix, ninst))
         restfiles = ""
         if compname.find('mpas') == 0 or compname == 'mali':
```

Comment:
@jonbob  - if I understand this correctly mpas now conforms to the standard naming convention and doesn't need to have a special block here?

Reply:
@jedwards4b - close but not completely. The MPAS components still have a different date string.

Reply:
that's unfortunate - thanks for clarifying.